product_filters/security/ir.model.access.csv

```diff
@@ -1,3 +1,4 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_product_price_ranges_manager,product_price_ranges_manager,model_product_price_ranges,group_product_price_ranges_manager,1,1,1,1
 access_product_price_ranges_user,product_price_ranges_user,model_product_price_ranges,,1,0,0,0
+access_product_brand_public,public.product.brand,product_brand.model_product_brand,base.group_public,1,0,0,0
```

Comment:
I was rethinking about this, since we no longer depend from the `website_product_brand` app and only from the `product_brand` app I think this is the right place to put the ACL

why?, because this is the app that publishes the model as readable on the web, not the `product_brand` app.

do you still think we should move the rule to a "backend only" app?